cmd/gitrelease/gitrelease.go

```diff
@@ -0,0 +1,462 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/base64"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"github.com/go-git/go-git/v5"
+	"github.com/go-git/go-git/v5/plumbing"
+	"github.com/go-git/go-git/v5/plumbing/object"
+	"github.com/supply-chain-tools/go-sandbox/gitkit"
+	"github.com/supply-chain-tools/go-sandbox/gitrelease"
+	"github.com/supply-chain-tools/go-sandbox/gitverify"
+	"log/slog"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"strings"
+)
+
+const usage = `Usage:
+    gitrelease [options]
+
+Options:
+    --keyfile      Path to SSH private key to use
+    --debug        Enable debug logging
+    -h, --help     Show help message`
+
```

Comment:
Should include the `--init` flag in usage instructions to avoid confusion, additionally, maybe prompt the user so that they don't need to rerun the command with a flag?

```suggestion
Options:
    --keyfile      Path to SSH private key to use
    --init         Initialise if no other tags should exist
    --debug        Enable debug logging
    -h, --help     Show help message`

```